src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -157,7 +157,9 @@ struct OperationViewModel {
             statusCodes.append(contentsOf: response.statusCodes)
         }
         for exception in exceptions {
-            statusCodes.append(contentsOf: exception.statusCodes)
+            for statusCode in exception.statusCodes where statusCode != "default" {
+                statusCodes.append(statusCode)
+            }
```

Comment:
Is `exceptions` a model object or a view model object? If it's a model object, it seems we could have a calculated property or helper method to handle this logic (I've seen it repeated, I think). If it's a view model, then I think we are already making sure that status codes is only numeric.